R/data_copy.R

```diff
@@ -130,7 +130,7 @@ spark_serialize_csv_scala <- function(sc, df, columns, repartition) {
     "sparklyr.Utils",
     "createDataFrameFromCsv",
     spark_context(sc),
-    tempfile,
+    paste("file://", tempfile, sep = ""),
```

Comment:
The Scala code [here](https://github.com/rstudio/sparklyr/blob/master/java/spark-1.5.2/utils.scala#L323) seems to be trimming the slashes, maybe we need to pass an `URI` on the Scala side or manipulate the string there?

Reply:
Where can I learn more about work to integrate with `Apache Arrow`?
Curious what use cases you have in plans.

Reply:
Javier, where can I learn more about work to integrate with `Apache Arrow`?
Curious what use cases you have in plans and mind.